controllers/storagecluster/prometheus.go

```diff
@@ -50,7 +50,7 @@ func getPrometheusRules(isExternal bool) (*monitoringv1.PrometheusRule, error) {
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      ruleName,
-			Namespace: ruleNamespace,
+			Namespace: os.Getenv(util.OperatorNamespaceEnvVar),
```

Comment:
Use `util.GetOperatorNamespace()` for safety. This could fail is env var is not set.

Reply:
We are already doing that in the main.go, So it should be safe. https://github.com/red-hat-storage/ocs-operator/blob/main/main.go#L120